composer.json

```diff
@@ -16,34 +16,38 @@
     ],
     "require": {
         "php": ">=7.2",
-        "spryker/container": "^1.1.0",
-        "symfony/event-dispatcher": "^3.0.0 || ^4.0.0",
-        "symfony/http-kernel": "^3.0.0 || ^4.0.0",
-        "symfony/routing": "^3.0.0 || ^4.0.0"
+        "spryker/container": "dev-beta/te-7001/te-7174/te-7175-create-beta-branch-for-spryker-container as 1.1.0",
+        "symfony/event-dispatcher": "^4.4.0 || ^5.0.0",
+        "symfony/http-kernel": "^4.4.0 || ^5.0.0",
+        "symfony/routing": "^4.4.0 || ^5.0.0"
     },
     "require-dev": {
-        "symfony/security": "^3.0.0 || ^4.0.0",
-        "symfony/config": "^3.0.0 || ^4.0.0",
-        "symfony/intl": "^3.0.0 || ^4.0.0",
-        "symfony/form": "^3.0.0 || ^4.0.0",
-        "symfony/browser-kit": "^3.0.0 || ^4.0.0",
-        "symfony/css-selector": "^3.0.0 || ^4.0.0",
-        "symfony/debug": "^3.0.0 || ^4.0.0",
-        "symfony/dom-crawler": "^3.0.0 || ^4.0.0",
-        "symfony/finder": "^3.0.0 || ^4.0.0",
-        "symfony/monolog-bridge": "^3.0.0 || ^4.0.0",
-        "symfony/options-resolver": "^3.0.0 || ^4.0.0",
-        "symfony/process": "^3.0.0 || ^4.0.0",
-        "symfony/serializer": "^3.0.0 || ^4.0.0",
-        "symfony/translation": "^3.0.0 || ^4.0.0",
-        "symfony/twig-bridge": "^3.2.0 || ^4.0.0",
-        "symfony/validator": "^3.0.0 || ^4.0.0",
-        "twig/twig": "~1.28|~2.0",
+        "symfony/security-core": "^4.2.7 < 4.4.0 || ^4.4.7 || ^5.0.0",
```

Comment:
the patches between 4.4.0 and 4.4.7 are incompatible? That doesnt make much sense, no?

Reply:
https://github.com/spryker/suite-nonsplit/blob/master/composer.json#L72

I just added v5 and dropped v3. V3 is not available for the splitted security components.